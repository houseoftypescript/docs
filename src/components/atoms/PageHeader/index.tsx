import Container from '@mui/material/Container';
import React from 'react';

type PageHeaderProps = {
  children?: React.ReactNode;
};

export const PageHeader: React.FC<PageHeaderProps> = ({ children = '' }) => {
  return (
    <header className="border-b">
      <Container className="py-12">{children}</Container>
    </header>
  );
};

PageHeader.displayName = 'PageHeader';
PageHeader.defaultProps = { children: '' };

export default PageHeader;
