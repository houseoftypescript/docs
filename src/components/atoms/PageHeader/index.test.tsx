import PageHeader from '.';
import { render } from '@testing-library/react';

describe('PageHeader', () => {
  test('render default', () => {
    const wrapper = render(<PageHeader />);
    expect(wrapper.container).toMatchSnapshot();
  });
});
