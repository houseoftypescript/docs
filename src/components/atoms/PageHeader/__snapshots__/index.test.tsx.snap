// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageHeader render default 1`] = `
<div>
  <header
    class="border-b"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg py-12 css-1oqqzyl-MuiContainer-root"
    />
  </header>
</div>
`;
